package.json

```diff
@@ -4,8 +4,8 @@
   "description": "Front-End Capstone Project",
   "main": "server/index.js",
   "scripts": {
-    "build": "webpack",
-    "serve": "nodemon",
+    "build": "webpack --config webpack.config.js -w",
```

Comment:
We have added `watch: true` to the webpack config, so we don't need the `-w` flag on the script. Also, we are using `webpack.config.js` so webpack intelligently finds it from the root directory, so we don't need that flag either.